src/AmbientSounds/ViewModels/ShellPageViewModel.cs

```diff
@@ -160,6 +160,11 @@ public void GoToScreensaver()
         _navigator.ToScreensaver();
     }
 
+    public async Task PlayRandomSoundAsync()
```

Comment:
Add [RelayCommand] above line 163. Change it to private

Reply:
I'm facing some problems with changing the task to private, something's complaining about that change :\ 
Adding RelayCommand worked though.

Reply:
what issue are you seeing?

Reply:
Nevermind, after adding the other change (click to command) it works.